cpp/basix/precompute.h

```diff
@@ -285,25 +279,26 @@ void pre_apply_matrix(
 
 /// @brief Apply a (precomputed) matrix to some transposed data.
 ///
+/// Computes \f$v = u M^{T}\f$.
```

Comment:
@mscroggs is this doc line correct?

Reply:
doesn't look right
```suggestion
/// Computes \f$v = M u\f$.
```

Reply:
Yes, but maybe it's clearer to write this:
```suggestion
/// Computes \f$v^{T} = Mu^{T}\f$ (or equivalently \f$v = u M^{T}\f$).
```